From c7345d78e01e276c97a12184fd014941c5a1b42e Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Mon, 25 Nov 2019 18:38:35 +0000
Subject: [PATCH] Kinetic lib prefix fix

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index be14020..984ad7d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,6 +19,8 @@ IF(NOT EXISTS "${CMAKE_SOURCE_DIR}/cmake/base.cmake")
   MESSAGE(FATAL_ERROR "\nPlease run the following command first:\ngit submodule update --init\n")
 ENDIF()
 
+set(CMAKE_INSTALL_LIBDIR "lib")
+
 INCLUDE(cmake/base.cmake)
 IF(CMAKE_VERSION VERSION_GREATER "3.0.0")
   CMAKE_POLICY(SET CMP0048 NEW)
-- 
2.17.1

